app/Nova/TravelAssignment.php

```diff
@@ -105,7 +106,17 @@ public function fields(NovaRequest $request): array
 
             BelongsTo::make('Trip', 'travel', Travel::class)
                 ->withoutTrashed()
-                ->rules('required', 'unique:travel_assignments,travel_id,NULL,id,user_id,'.$request->user)
+                ->rules(
+                    'required',
+                    'unique:travel_assignments,travel_id,NULL,id,user_id,'.$request->user,
```

Comment:
Is there supposed to be a `->id` here?

Reply:
No, `user` here is referring to a request attribute, not the authenticated user.